Source/WebCore/css/CSSValueList.cpp

```diff
@@ -211,11 +211,9 @@ bool CSSValueContainingVector::hasValue(CSSValueID otherValue) const
 
 CSSValueListBuilder CSSValueContainingVector::copyValues() const
 {
-    CSSValueListBuilder builder;
-    builder.reserveInitialCapacity(size());
-    for (auto& value : *this)
-        builder.uncheckedAppend(const_cast<CSSValue&>(value));
-    return builder;
+    return WTF::map<4>(*this, [](auto& value) -> Ref<CSSValue> {
```

Comment:
The idea is that `CSSValueListBuilder` could change its inline capacity in the future. The hardcoded 4 here doesn’t seem great, especially if it will make working but inefficient code in case of mismatch. I suppose it’s OK if compilation will just fail,

Reply:
It fails to build at the moment:
```
Source/WebCore/css/CSSValueList.cpp:214:12: error: no viable conversion from returned value of type 'Vector<[...], 0UL aka 0, [2 * ...]>' to function return type 'Vector<[...], 4, [2 * ...]>'
```

That said, it is probably better to use a global constant to be safe. Nothing stops someones from updating the Vector class to convert silently.